### ISSUE TITLE
Does plugin library now support cmi stats?

### ISSUE THREAD

**Issue by FrozenBeard** [Staartvin/Autorank-2#701]
I noticed you support cmi now in plugin library, does this mean we can use cmi stats and not have to use statz?

**Comment by Staartvin** [comment]
That's not supported yet, but I'll make this a feature request.

**Comment by Staartvin** [comment]
What exactly do you want to be imported from CMI? I'm looking at the page but can't really find any statistics?

**Comment by FrozenBeard** [comment]
Cmi has its own stats+autoranking, he pulls stats directly from the world folder. There are too many problems with stats3/5 and statz I just need something reliable, so he's got /cmi stats which has a nice gui for showing stats, I would like autorank to be able to read his stats, his autoranking is a little limited and there's a bug he's not fixed. Stats and autoranking is the only thing holding me back for 1.13+ it's driving me nuts

**Comment by Staartvin** [comment]
Let me check if there is an API for it on his side. Otherwise I'll resort to loading it from the world folder myself.

**Comment by FrozenBeard** [comment]
Awesome. If you have CMI, check out his gui for displaying stats, I thought. As the world folder stats is JSON, I should be able to write a simple enough script to convert my stats3 I just need to move my data somewhere that's going to be reliable!

**Comment by Staartvin** [comment]
Unfortunately I cant check the functionality as I don't have CMI, but I'll try to test it as best as I can.

**Comment by FrozenBeard** [comment]
I can send you my copy? Its for extending his plugin so should be fine as its not being used on a server.

**Comment by Staartvin** [comment]
Yeah, that would be nice. Send it on Discord or to my email address, whatever you prefer.

**Comment by Staartvin** [comment]
@FrozenBeard Did you send it? I can't seem to find it.
